src/components/ui/Chat/MessageList/formatMessageList.tsx

```diff
@@ -0,0 +1,114 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React from 'react';
+
+import DateHeader from './DateHeader';
+import ChatBubble from '../ChatBubble';
+import { PopOverItemProps } from '../../PopOver/PopOverItem';
+
+export interface MessageProps {
+  /* The displayed text of the message sent. */
+  content: string;
+  /* The unique identifier of the message. */
+  messageId: string;
+  /* The timestamp when the message was originally sent by the sender. */
+  createdTimestamp: string;
+  /* The timestamp of the last time the message was edited. */
+  lastUpdatedTimestamp: string;
+  /* Determines if a message was redacted (deleted) by a user. */
+  redacted: boolean;
+  /* The unique identifier of the channel the message was sent in. */
+  channelArn?: string;
+  sender: {
+    /* The display name of the sender. */
+    name: string;
+    /* The unique identifier of the sender. */
+    arn: string;
+  }
```

Comment:
This data structure matches that of the messages API from the chat service. I thought it would be easy to plug and play this way. I could flatten it which would only require the mildest of data manipulation in the app.